make/modules/jdk.accessibility/Launcher.gmk

```diff
@@ -43,7 +43,7 @@ ifeq ($(call isTargetOs, windows), true)
       CFLAGS := $(filter-out -Zc:wchar_t-, $(CFLAGS_JDKEXE)) -Zc:wchar_t \
           -analyze- -Od -Gd -D_WINDOWS \
           -D_UNICODE -DUNICODE -RTC1 -EHsc, \
-      DISABLED_WARNINGS_microsoft := 4267 4996, \
+      DISABLED_WARNINGS_microsoft_jabswitch.cpp := 4267 4996, \
```

Comment:
This lib only has one cpp file, so this change doesn't seem that useful.

Reply:
Heh, I did not notice that.

I'd still argue that this is a useful change. If some other file were ever to be added, it will not be covered by the blanked disabled warnings, and -- perhaps more important -- it is clear that these warnings can be addressed just by looking at a single file. This information has drastically increased the likelihood that a developer picks up the issue and actually resolves the warnings, as we have seen in other areas where I've done this "warnings-per-file" transition.

Reply:
I suspect there's already a "fix disabled warnings in jdk.accessibility" .. can't we just fix that instead ?
I found it : https://bugs.openjdk.org/browse/JDK-8074847
Let me see what I can do about it.

Reply:
but I'll approve this anyway .. no harm